parts/kubernetesmasteraddons-heapster-deployment.yaml

```diff
@@ -40,11 +40,9 @@ spec:
         - "--extra-cpu=0.5m"
         - "--memory=140Mi"
         - "--extra-memory=4Mi"
-        - "--threshold=5"
```

Comment:
We rolled the addon-resizer back across the board currently so this change is irrelevant -- #1076